go/libraries/doltcore/sqle/docsTable/dolt_docs_table.go

```diff
@@ -0,0 +1,145 @@
+// Copyright 2020 Dolthub, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package docsTable
```

Comment:
Put this in dtables with other persisted system tables (dolt_query_catalog), not in a new package

Reply:
Blocked on this because of import loop stemming from status table :/

Reply:
I'm fine with it being separate for now, not a huge deal

Reply:
But you should try again at the end once you do everything else, it might unknot itself.